src/main/java/com/codeus/winter/config/BeanDefinition.java

```diff
@@ -0,0 +1,133 @@
+package com.codeus.winter.config;
+
+import javax.annotation.Nullable;
+
+public interface BeanDefinition {
+    /**
+     * Scope identifier for the standard singleton scope: {@value}.
+     */
+    String SCOPE_SINGLETON = "SCOPE_SINGLETON";
```

Comment:
I suppose it should be enum with name Scope and constants SINGLETON and PROTOTYPE

Reply:
Fun fact: in Spring, these constants are also in this BeanDefinition class:
<img width="387" alt="Screenshot 2024-11-05 at 09 06 35" src="https://github.com/user-attachments/assets/3f3496c6-ada8-4bb8-873e-3a8f0a648946">
But I agree with you, enum would be a more convenient place.

Reply:
I fixed it with Enum